azkaban-exec-server/src/main/java/azkaban/execapp/FlowRunnerManager.java

```diff
@@ -1055,8 +1055,14 @@ private void pollExecution() {
         }
       } else if (this.pollingCriteria.shouldPoll()) {
         try {
-          final int execId = FlowRunnerManager.this.executorLoader
-              .selectAndUpdateExecution(this.executorId, FlowRunnerManager.this.active);
+          final int execId;
+          if (FlowRunnerManager.this.azkabanProps.getBoolean(ConfigurationKeys.AZKABAN_POLLING_LOCK_ENABLED, false)) {
```

Comment:
Can we add a comment here as we have conditional lock based and regular execution now?

Reply:
Added boolean in case we want to revert. We intend to have only lock based polling after the change is tested on clusters.